.jenkins

```diff
@@ -72,6 +72,36 @@ pipeline {
                         '''
                     }
                 }
+                stage('SYCL') {
+                    agent {
+                        dockerfile {
+                            filename "Dockerfile.sycl"
+                            dir "docker"
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache'
```

Comment:
What's the point of mounting it if you don't use `ccache`?

Reply:
Good point - I'll add